tools/fuchsia/fidl/gen_response_file.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3.8
```

Comment:
I haven't look too closely at how fidl is setup, i'm assuming this is mostly using the same logic as in the fuchsia tree? please let me know if you'd like me to review the fidl stuff more carefully.

Reply:
Yes, almost all of the logic was copied (and minimally tailored) from fuchsia.git. More on this in my other response, below.

I don't think you need to review the details. If you think this is helpful: you could try a recursive diff of the additional files in `//flutter/tools/fuchsia/*/*` with Fuchsia's `//build/*/` directory to confirm they closely match. That could give you more confidence.